extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/namespace/JDBCExtractionNamespace.java

```diff
@@ -55,10 +55,8 @@ public JDBCExtractionNamespace(
       final MetadataStorageConnectorConfig connectorConfig,
       @NotNull @JsonProperty(value = "table", required = true)
       final String table,
-      @NotNull @JsonProperty(value = "keyColumn", required = true)
-      final String keyColumn,
-      @NotNull @JsonProperty(value = "valueColumn", required = true)
```

Comment:
Ah, I see we already have this in `DEFAULT_MAPNAME`. Let's use that for this purpose.